astTraversal/result.go

```diff
@@ -50,6 +50,8 @@ type Result struct {
 
 	StructFieldValidationTags ValidationTagMap
 
+	StructFieldValidationRequired ValidationRequiredMap
```

Comment:
Docstring? Also for the other bits you've added (I'm aware I'm guilty of not adding this as well)

Reply:
Agreed I have now added them.